### ISSUE TITLE
need dependency on axios

### ISSUE THREAD

**Issue by xenoterracide** [new10com/axios-logger#8]
with yarn 2 PnP, modules cant require/import  other modules unless they are a dependency, so it won't pick up axios just because I imported it.
```.yarn/cache/@new10com-axios-logger-npm-0.1.4-6f2a3cd08a-f7c3d25b86.zip/node_modules/@new10com/axios-logger/dist/src/axios-logger.d.ts:1:51 - error TS2307: Cannot find module 'axios' or its corresponding type declarations.

1 import { AxiosRequestConfig, AxiosResponse } from 'axios';
```

maybe worth saying on version 0.20 I have this issue https://github.com/axios/axios/issues/3219

**Comment by azakordonets** [comment]
@xenoterracide sorry not sure i'm following you - do you suggest to add `axios` to `dependencies` of this library, or to upgrade `axios` library to 0.21 version ?

**Comment by xenoterracide** [comment]
yes, both. or maybe downgrade it to 0.19.2, I notice a lot of other libraries using 0.19...

**Comment by azakordonets** [comment]
@xenoterracide fixed in v.0.1..6 . Let me know if it's still a problem.

**Comment by xenoterracide** [comment]
hmm... I'm not able to reproduce it, but then I'm not working on that branch... and I lamely didn't post enough about what I was doing. however it looks like you have that dependency in `devDependencies` when it should be in `dependencies` also, I'm ok with a dependency on `^0.19.2` I just wanted to be able to do earlier than 0.20

**Comment by azakordonets** [comment]
@xenoterracide added deps in 0.1.7. Let me know if it's ok to close this ticket

**Comment by xenoterracide** [comment]
better, but you still might want to make the dep *^*0.19.2 so if someone wants to use 0.21.x they can without ending up with a second version. I'll leave that up to you, but otherwise looks fine.

**Comment by azakordonets** [comment]
You're right, i missed it. Corrected in v0.1.8. Thanks!

**Comment by xenoterracide** [comment]
looks good, thanks for the work
